datasafe-rest-impl/2.startDockerImageWithLocalFilesystem.sh

```diff
@@ -0,0 +1,37 @@
+#
+# trap with sigint because container can not be stopped from command line otherwise
+#
+
+trap stop SIGINT
+
+function stop() {
+	echo stop
+	docker stop -t 0 datasafe-rest-test
+}
+
+mkdir target/ROOT_BUCKET
+
+docker run 										                                        \
+	--rm 											                                    \
+	--name datasafe-rest-test								                            \
+	-e JWT_SECRET=your_personal_secret                                               	\
+    -e DEFAULT_USER=your_user_to_come_to_the_rest_api 							        \
+    -e DEFAULT_PASSWORD=password_for_rest_api 								            \
+	-p 8080:8080 										                                \
+	-e ENABLE_FRONTEND=true 								                            \
+	-e USE_FILESYSTEM=/usr/app/ROOT_BUCKET                                              \
+	-v $(pwd)/target/ROOT_BUCKET:/usr/app/ROOT_BUCKET                                   \
+	datasafe-rest-test:latest > startDockerImage.log 2>&1 &
```

Comment:
Why one needs log file from container that is mostly for demo?

Reply:
Actually specially for someone new to the project this is very helpfull when he first plays around with the code.

Reply:
Well pbcopy is present only on MacOS, so this script will fail for non MacOS users. Also he will see logs in terminal too

Reply:
ok, logs come to sysout with -it.
pbcopy now only done if available